ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -18,6 +18,24 @@
 
 ### Single Zone Units ###
 
+AHU_SS_HWSC_ETM_SFM_FDPSM_SFVSC_EFVSC_SDM_EDM_WDPM_VSC:
```

Comment:
Again, your AHU does not have any ECON modeled and is therefore incomplete. Please confirm if this is an AHU or DOAS